src/main/java/org/cascadebot/cascadebot/commands/moderation/PurgeCommand.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.commands.moderation;
+
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.entities.Member;
+import org.cascadebot.cascadebot.commandmeta.*;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.utils.PurgeUtils;
+
+import java.util.Set;
+
+public class PurgeCommand implements ICommandMain {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        if (context.getArgs().length < 1) {
+            context.getUIMessaging().replyUsage();
+            return;
+        }
+        try {
+            context.getArgAsInteger(0);
+        } catch (NumberFormatException error) {
```

Comment:
You should use the `context.isArgInteger` for all the commands where you call `context.getArgAsInteger`

Reply:
okay